handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -117,4 +105,31 @@ public JdkAlpnApplicationProtocolNegotiator(ProtocolSelectorFactory selectorFact
             ProtocolSelectionListenerFactory listenerFactory, String... protocols) {
         super(ALPN_WRAPPER, selectorFactory, listenerFactory, protocols);
     }
+
+    private static final class FailureWrapper implements SslEngineWrapperFactory {
+        @Override
+        public SSLEngine wrapSslEngine(SSLEngine engine, JdkApplicationProtocolNegotiator applicationNegotiator,
+                                       boolean isServer) {
+            throw new RuntimeException("ALPN unsupported. Is your classpatch configured correctly?"
+                    + " For Conscrypt, add the appropriate Conscrypt JAR to classpath."
```

Comment:
in reference to https://github.com/netty/netty/pull/6271/files#r104705400 ... should this also say "...and also set the security provider to use Conscrypt" ?